docs/services/firecrest-ui.md

```diff
@@ -0,0 +1,55 @@
+[](){#ref-firecrest-ui}
+# FirecREST UI
+
+FirecREST UI is a web application designed to provide essential functionalities for interacting with [FirecREST API v2 backend](https://docs.cscs.ch/services/firecrest/).
```

Comment:
I would suggest to spell 'UI' at least once as User Interface, e.g. and take a user point of view, i.e. what it provides

Firecrest User Interface (UI) is a web application that provides a convenient access to information, data and functionalities for a particular cluster. Its implementation is based on the [FirecREST API v2 backend][ref-firecrest].